compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -351,6 +351,7 @@
     private static final String GENERATED_ERROR_VAR = "$error$";
     private static final String HAS_KEY = "hasKey";
     private static final String CREATE_RECORD_VALUE = "createRecordFromMap";
+    private static final String METHOD_INIT = "$init$";
```

Comment:
This should be defined as constant in Names. Can u check?

Reply:
Addressed with https://github.com/ballerina-platform/ballerina-lang/pull/32838/commits/f202d01ccb60a0671bfcc61d3c9387e9e1b16e4e